.github/actions/override-sdk-version/action.yaml

```diff
@@ -0,0 +1,30 @@
+name: Override SDK tag
```

Comment:
Is this action used in this PR or was does it need to be shifted over to https://github.com/LF-Decentralized-Trust-labs/paladin/pull/846 along with the other SDK override changes?